library/src/camera_impl.cxx

```diff
@@ -0,0 +1,256 @@
+#include "camera_impl.h"
+
+#include <vtkCamera.h>
+#include <vtkMatrix4x4.h>
+#include <vtkRenderer.h>
+
+namespace f3d::detail
+{
+class camera_impl::internals
+{
+public:
+  vtkRenderer* VTKRenderer = nullptr;
+  vtkNew<vtkCamera> DefaultVTKCamera;
+};
+
+//----------------------------------------------------------------------------
+camera_impl::camera_impl()
+  : Internals(new camera_impl::internals())
```

Comment:
`Internals(std::make_unique<camera_impl::internals>())`?

Reply:
done in another pr: #340